README.md

```diff
@@ -15,6 +16,12 @@ a Geographic Information System used for geospatial data management and
 analysis, image processing, graphics/map production, spatial modeling, and
 visualization.
 
+## Jupyter Notebook
+Launch this repository in binder and experiment with PyGRASS in Jupyter Notebooks by clinking the button below.
+
+[![Binder](https://camo.githubusercontent.com/581c077bdbc6ca6899c86d0acc6145ae85e9d80e6f805a1071793dbe48917982/68747470733a2f2f6d7962696e6465722e6f72672f62616467655f6c6f676f2e737667)](https://mybinder.org/v2/gh/chaedri/grass/binder?urlpath=lab/tree/doc%2Fnotebooks%2Fexample_notebook.ipynb)
```

Comment:
You need to generate the badge for this repo, not your fork.

Reply:
Oops! Yes, I'll fix the link.

As for PyGRASS, I meant the whole Python API (I was still figuring out the difference).